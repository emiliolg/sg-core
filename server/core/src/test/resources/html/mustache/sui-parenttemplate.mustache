<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views" sg-params="content : Html">
<head>
    <title>Parent Template</title>
</head>
<body>

<h1>Include</h1>

{{>html/mustache/sui-subtemplate}}

{{#sg-include}}content{{/sg-include}}

</body>
</html>
