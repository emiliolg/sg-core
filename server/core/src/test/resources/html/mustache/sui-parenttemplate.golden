<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views" sg-params="content : Html">
<head>
    <title>Parent Template</title>
</head>
<body>

<h1>Include</h1>

<h2>Child Template</h2>

<h3>Child Included from parameter Template</h3>

</body>
</html>
